packages/connectkit/src/utils/index.ts

```diff
@@ -110,6 +110,8 @@ const isMetaMask = () => {
   const isTokenary = Boolean(ethereum.isTokenary);
   if (isTokenary) return false;
 
+  if (isPhantom()) return false;
```

Comment:
@nheingit We require this check to stop wallets from overriding metamask so they can be treated as their own connector